### PR TITLE
Create `FAQ.md`

### PR DESCRIPTION
**Pull request by MaxDesiatov**
This is a document we could redirect people to with questions about our layout system, naming, and history of the project. Feel free to expand this with more relevant topics.

### CODE REVIEW

**Review by j-f1 — DISMISSED** [review]

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+size and performance overhead. With HTML/CSS we can rely on what's already included in the browser and has been tested and polished
+by hundreds of engineers over the decades of browser development.
+
+Additionally, we can rely on optimized CSS layout algorithms where possible. This also unlocks more use-cases for Tokamak, such as
+[static HTML generation](https://github.com/TokamakUI/TokamakPublish) and [server-side rendering](https://github.com/TokamakUI/TokamakVapor).
+
+At the same time, Tomakak has [a new layout system in development](https://github.com/TokamakUI/Tokamak/pull/472) that accesses
+DOM directly for layout calculations, bypassing CSS for a lot (or potentially all) of its algorithms.
+
+## Does the word Tokamak mean anything? Why is it called this way?
```
Comment (carson-katri) [inline docs/FAQ.md:16]:
```suggestion
## Does the word Tokamak mean anything? Why is it called this?
```

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+HTML/CSS has a benefit of built-in accessibility support. Other rendering systems in the browser (such as Canvas or WebGL/WebGPU)
+that bypass HTML/CSS would have to reimplement accessibility from scratch, with all the downsides of increased binary
+size and performance overhead. With HTML/CSS we can rely on what's already included in the browser and has been tested and polished
+by hundreds of engineers over the decades of browser development.
+
+Additionally, we can rely on optimized CSS layout algorithms where possible. This also unlocks more use-cases for Tokamak, such as
+[static HTML generation](https://github.com/TokamakUI/TokamakPublish) and [server-side rendering](https://github.com/TokamakUI/TokamakVapor).
+
+At the same time, Tomakak has [a new layout system in development](https://github.com/TokamakUI/Tokamak/pull/472) that accesses
```
Comment (carson-katri) [inline docs/FAQ.md:13]:
```suggestion
At the same time, Tokamak has [a new layout system in development](https://github.com/TokamakUI/Tokamak/pull/472) that accesses
```

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+## Does the word Tokamak mean anything? Why is it called this way?
+
+The project was originally inspired by [React](https://reactjs.org), which utilizes a model of an atom in its logo,
+apparently as a reference to nuclear reactors. [Токамак](https://en.wikipedia.org/wiki/Tokamak) is a nuclear fusion reactor, and 
+the word itself is roughly an abbreviation of "**to**roidal **cha**mber with **ma**gnetic **c**oils".
+
+##  What's the history behind it?
+
+The first commit to this project was made in September 2018, 9 months before SwiftUI was publicly announced. Original maintainer of
```
Comment (carson-katri) [inline docs/FAQ.md:24]:
```suggestion
The first commit to this project was made in September 2018, 9 months before SwiftUI was publicly announced. The original maintainer of
```

**Review by carson-katri — CHANGES_REQUESTED** [review]
Thanks for writing this up, I had a couple of nitpicks but other than that it looks great.

File: docs/FAQ.md (hunk 2 above)
Comment (MaxDesiatov) [reply docs/FAQ.md:13]: Haha, amazing catch, thank you! 😅

**Review by carson-katri — APPROVED** [review]

**Pull request merged**